clang-tools-extra/test/clang-tidy/checkers/modernize/use-default-member-init.cpp

```diff
@@ -518,3 +518,22 @@ class ArrayBraceInitMultipleValues {
 };
 
 } // namespace PR63285
+
+namespace PR122480 {
+
+  static int STATIC_VAL = 23;
+  static constexpr const char* CONSTEXPR_REF = "Static";
```

Comment:
remove static, to check only first matcher.
```suggestion
  constexpr const char* CONSTEXPR_REF = "Static";
```